### PR TITLE
docs: Update and remove deprecated documentation, including new publi…

### PR DESCRIPTION
**Pull request by JackHowa**
- Emphasize rebasing after merging into new branch and publishing https://github.com/WPMedia/fusion-news-theme-blocks/pull/596
- Remove outdated docs
- Referred to steps docs in fusion news theme step-by-step guide

### CODE REVIEW

File: News Theme Development.md [hunk 1]
```diff
@@ -54,14 +42,12 @@ The news-theme-css repo is located at: <https://github.com/WPMedia/news-theme-cs
...
 
 While the style guide will provide a good overview, it's advisable to review the Sass files as well to have a proper understanding of its usage. For example, if you look in scss/\_variables.scss you will see variables such as `$primary-font-family` and `$secondary-font-family`. These variables use the Sass notation `!default` at the end so that they can be over-ridden in the themes file, `blocks.json`, located in the fusion-news-theme repository. (We will talk about `blocks.json` more in the fusion-news-theme section.). Also, if you look in scss/\_breakpoints.scss you will see Sass maps for breakpoints and spacing. These maps are leveraged with the Sass function, `map-get`, in component Sass code to set spacing and media queries.
 
 news-theme-css is provided to the other parts of the system as an NPM package. For more information on the build and publishing process for this repository, please see the readme.md for the project: <https://github.com/WPMedia/news-theme-css/blob/stable/readme.md>
 
-**Note:** When publishing, you will need a .npmrc file that gives you access to the private NPM repo. Reach out to a team member to get this.
```
Comment (beltrancaliz) [inline News Theme Development.md:63]: Maybe we should reword this just to make sure that clients have it when they create a bundle (even if we don't go through the steps for creating the token here)

File: News Theme Development.md [hunk 2]
```diff
@@ -168,26 +129,15 @@ git checkout -b PEN-[jira ticket num]-[brief description of feature]
 
 #### How To Publish
 
-Before publishing, make sure a Pull Request has been made and merged from the `canary` branch against the `beta` branch. This PR should only include features that belong to the current release, so make sure to merge as soon as the features for the release have been completed to avoid including further features from `canary` belonging to a later release.
-
----
+Merge into `canary` branch to publish to canary tag. Please reach out to arc block maintainers to talk about publishing into `beta` or `stable` or other tags.
```
Comment (beltrancaliz) [inline News Theme Development.md]: add `rc`?

**Review by beltrancaliz — CHANGES_REQUESTED** [review]
Added a few suggestions

File: News Theme Development.md (hunk 1 above)
Comment (JackHowa) [reply News Theme Development.md:63]: good idea

**Comment by JackHowa** [comment]
@beltrancaliz updated -- thanks for review

File: News Theme Development.md (hunk 1 above)
Comment (JackHowa) [reply News Theme Development.md:63]: updated

File: News Theme Development.md (hunk 2 above)
Comment (JackHowa) [reply News Theme Development.md]: yep updated

**Pull request merged**